### DISCUSSION TITLE
ProblemDetail 꼭 사용해야하나요?

### DISCUSSION THREAD

**Discussion by zeus6768** [woowacourse-teams/2024-code-zap#237] [이야기해봐요]
필요성을 잘 못 느끼겠어요. 커스텀 예외 클래스를 사용하는 것에 비해 장점이 무엇인가요?

**Comment by jminkkk** [comment]
1. 스프링에서 제공해주기 때문에 따로 커스텀 예외 클래스 만드는 등 관리하지 않아도 된다.
2. [RFC 9457 규약](https://www.rfc-editor.org/rfc/rfc9457.html#name-introduction)을 따르는 응답이 된다.
